# SC-ST-Characteristics

Liver cancer is the leading cause of cancer deaths worldwide, with a five-year relative survival rate of only 22%[1]. Hepatocellular carcinoma (HCC) is the most common type of primary liver cancer and is typically induced by hepatitis B or C virus infection, alcoholism, and cirrhosis[2]. Current therapeutic options for HCC have limited efficacy, largely due to the intratumoral heterogeneity of HCC, which contributes to its aggressiveness, recurrence, drug resistance, and immune escape[3]. This heterogeneity arises from the interactions between cancer cells and various cell subpopulations within their microenvironment, resulting in distinct biological functions across different regions of the tumor[4-7]. Additionally, the spatial organization of the tumor provides a more intuitive way to quantify the various regions within the microenvironment[8]. When combined with pathological analysis of tumor sections, this can reveal the biological characteristics of different tumor regions. The tumor infiltration leading edge (LE) is composed of a layer of cells at the tumor's periphery, which exhibit distinct phenotypic states and microenvironmental characteristics compared to the tumor core (TC) region. The LE is the interface between the tumor and normal cells and plays a crucial role in tumor invasion[9]. However, there is still a lack of understanding of the activation mechanisms of LE regions within the spatial organization of tumors. Further exploration of the microenvironmental composition of the LE region in HCC will enhance our understanding of the mechanisms underlying HCC initiation and progression, and provide a new theoretical foundation for the development of personalized therapeutic strategies.
Previous studies have primarily focused on low-throughput or single-level analyses, making it challenging to comprehensively uncover the relationship between the HCC tumor immune microenvironment (TIME) and tumor biological behaviors[10-12]. With the advent of single-cell transcriptomic technology, we can now explore the interplay between the TIME environment and tumor biological functions at the cellular resolution level. However, single-cell technology has limitations in analyzing the spatial organization of tumors, which spatial transcriptomics is designed to address. Spatial transcriptomics technology enables the study of cell cluster expression with subcellular spatial resolution, aiding our understanding of their arrangement and distribution within tissues and organs, and allowing for the integration of rich molecular expression data into pathology analysis[13,14]. Therefore, combining single-cell transcriptome and spatial transcriptomics technologies allows us to simultaneously investigate tumor heterogeneity and spatial organization and structure from both cellular and tissue levels, and to explore patterns of molecular changes during HCC development from multiple perspectives. In this study, we integrated single-cell transcriptome sequencing with spatial transcriptomics to characterize the composition and spatial regions of TIME, thereby revealing the intratumor heterogeneity of HCC. We found that the leading edge (LE) region of the tumor was significantly enriched for TIME-ID and TIME-IR phenotypes and exhibited ligand-receptor interactions involved in anti-immune and stromal generation signaling pathways, suggesting a more aggressive nature of this region. Furthermore, signaling expression in the LE region was significantly correlated with prognosis and associated with clinical indicators such as tumor stage and metastasis. The genes VTN, FN1, CD99, and APP evaded immune system surveillance by constructing the cellular matrix skeleton and activating pathways such as PI3K/AKT, TGF-β, and Wnt/β-catenin, thereby enhancing the invasiveness of cancer cells[15-20]. Utilizing deep neural network (DNN) models, we were able to rapidly and accurately predict the distribution of TIME isoforms and the composition of spatial structural regions within the tumor's spatial organization. Additionally, we employed RNA velocity to infer the evolutionary patterns of TIME and spatial structure within tumors. By combining this with in-silico perturbation and sensitivity analysis algorithms, we identified potential therapeutic candidates with lower half maximal inhibitory concentration (IC50) values and higher Active Agent Concentration (AAC) values, which disrupt the TC to LE signaling flow, thereby reversing the tumor's developmental trajectory.
In conclusion, our study reveals the intra-tumor heterogeneity of HCC from both cellular and tissue perspectives. We have identified spatial regions and marker genes that are associated with tumor aggressiveness, and screened potential therapeutic agents accordingly. These findings provide novel research insights for a deeper understanding of the occurrence and development of HCC.
